benchs/bench_ivfflat_cuvs.py

```diff
@@ -43,8 +43,8 @@ def aa(*args, **kwargs):
    help='whether to benchmark add operation on GPU index')
 aa('--bm_search', default=True,
    help='whether to benchmark search operation on GPU index')
-aa('--raft_only', default=False, action='store_true',
```

Comment:
Have you run this before and after the change to determine the change (if any) in performance?